lib/jobs/install-os.js

```diff
@@ -95,13 +95,21 @@ function installOsJobFactory(
                 }
             });
         }
-
         if (this.options.users) {
             _.forEach(this.options.users, function(user) {
                 assert.string(user.name);
                 assert.string(user.password);
                 assert.number(user.uid);
             });
+            this.context.users = this.options.users;
+        }
+        if (this.options.rootPassword || this.options.rootSshKey) {
```

Comment:
Meanwhile, add a reminder to you, we plan to change the `sshKey` and `rootSshKey` to plural, this is the story (https://www.pivotaltracker.com/projects/1492892/stories/117029201). If your code can take consideration both the singular and plural cases, then we can reduce associated change when doing former story.